src/main/host/tracker.h

```diff
@@ -25,6 +20,11 @@ enum _LogInfoFlags {
     LOG_INFO_FLAGS_RAM = 1<<2,
 };
 
+#include "main/core/support/definitions.h"
+#include "main/host/protocol.h"
+#include "main/routing/packet.h"
+#include "support/logger/log_level.h"
```

Comment:
Should the `LogInfoFlags` enum be moved to a separate file so that it's not before the include statements?

Reply:
Done. Oops, yeah meant to circle back and fix that.